openssl/src/ssl/tests/mod.rs

```diff
@@ -50,18 +51,28 @@ struct Server {
 
 impl Server {
     fn spawn(args: &[&str], input: Option<Box<FnMut(ChildStdin) + Send>>) -> (Server, SocketAddr) {
+        static CERT: &'static [u8] = include_bytes!("../../../test/cert.pem");
+        static KEY: &'static [u8] = include_bytes!("../../../test/key.pem");
+
+        let mut me = env::current_exe().unwrap();
+        me.pop();
+        let cert = me.join("cert.pem");
+        let key = me.join("key.pem");
+
+        File::create(&cert).unwrap().write_all(CERT).unwrap();
```

Comment:
You ok picking that up as a dep? I figured this was as good a place as any without picking up new dependencies.